pmaweb/urls.py

```diff
@@ -357,15 +357,15 @@
         r'^downloads/phpMyAdmin-latest-'
         r'(?P<flavor>all-languages|english)'
         r'(?P<extension>\.zip|\.tar\.gz|\.tar\.xz|\.7z|\.tar\.bz2)'
-        r'(?P<checksum>\.asc|\.sha256)?$',
+        r'(?P<checksum>\.asc|\.sha256|\.sha1)?$',
```

Comment:
I don't see much reason adding sha1 checksum which is considered insecure, see https://github.com/phpmyadmin/website/issues/59

Reply:
@nijel Because all other versions do support sha1
See: https://files.phpmyadmin.net/phpMyAdmin/4.8.3/phpMyAdmin-4.8.3-english.tar.gz.sha1

Reply:
maybe it's time to remove it from future releases there :-). It's not used in the web anymore, so there is no need to generate them...

Reply:
Yes, another PR to remove them is maybe a better idea :)
What about sha1 in https://www.phpmyadmin.net/packages.json ?

Reply:
Ah, it probably needs to be there, so we need the sha1 files as well (it can be computed on the server, but it's safer to have it uploaded).

Reply:
@nijel Ok, so this PR can be merged ? or do I need to change something else ?